cypress/integration/Input/Can_set_a_value/index.js

```diff
@@ -0,0 +1,19 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a Input with no text is rendered', () => {
+    cy.visitStory('Input', 'Default')
```

Comment:
Above I saw that we were prefixing the stories used for integration tests with `Testing`, I'd expect that to be uniform (so everywhere the same naming strategy). I'm not sure if we have a separate user-facing storybook, but I'd prefer a clear and consistent separation between user facing stories and testing stories.